examples/p4est_2d_dgsem/elixir_navierstokes_lid_driven_cavity.jl

```diff
@@ -0,0 +1,82 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+# TODO: parabolic; unify names of these accessor functions
```

Comment:
Is this on the TODO list for parabolic solvers?

Reply:
Yup - filed in a separate issue https://github.com/trixi-framework/Trixi.jl/issues/1503 and added it to the parabolic issue https://github.com/trixi-framework/Trixi.jl/issues/1147